src/libs/checkForUpdates.ts

```diff
@@ -0,0 +1,19 @@
+const UPDATE_INTERVAL = 1000 * 60 * 60 * 8;
+
+type CheckForUpdatesParams = {
+    update: () => void;
+    init?: () => void;
+};
+
+function checkForUpdates(platformSpecificUpdater: CheckForUpdatesParams) {
+    if (typeof platformSpecificUpdater.init === 'function') {
```

Comment:
```suggestion
    if (_.isFunction(platformSpecificUpdater.init)) {
```

As much as I'd like to use native functions everywhere, let's stick to `_.isFunction()` and not change this

Reply:
Hi @arosiclair 
from https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#migration-guidelines
> Deprecate the usage of underscore. Use vanilla methods from JS instead. Only use lodash when there is no easy vanilla alternative (eg. lodashMerge).

We should not use underscore anymore

Reply:
For context, that was discussed [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1687477811950459) @arosiclair

Reply:
Awesome that's good news. Let's keep it in 👍